internal/app/pass/vault/vault_test.go

```diff
@@ -0,0 +1,77 @@
+package vault
+
+import (
+	"bytes"
+	"crypto/aes"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var (
+	filepath      = "./test-vault.vlt"
+	masterKey     = []byte("0123456789ABCDEF")
+	expectedVault *Vault
+)
+
+func init() {
+	expectedVault = NewVault(filepath, masterKey)
+	iv := bytes.Repeat([]byte{0}, aes.BlockSize)
+	expectedVault.encryptionIvReader = bytes.NewBuffer(iv)
+
+	expectedVault.AddEntry(Entry{BinaryPath: "test1", Secrets: map[string]string{"foo1": "bar1", "foo2": "bar2"}})
+	expectedVault.AddEntry(Entry{BinaryPath: "test2", Secrets: map[string]string{"foo1": "bar1", "foo2": "bar2"}})
```

Comment:
Maybe you can change the `Secrets` of `test2` to ensure that you get the right entries in your tests? (same bellow)